resources/qml/Dialogs/RenameDialog.qml

```diff
@@ -55,32 +59,58 @@ UM.Dialog
             id: nameField
             width: parent.width
             text: base.object
+            placeholderText: base.objectPlaceholder
+            placeholderTextColor: UM.Theme.getColor("text_field_text_disabled")
             maximumLength: 40
             selectByMouse: true
             onTextChanged: base.textChanged(text)
         }
 
+        // spacer
+        Rectangle
+        {
+            color: base.backgroundColor
+            height: UM.Theme.getSize("wide_margin").height
+            width: height
+        }
+
+        Row
```

Comment:
It would be nice to have a comment here about what extraInfo is. Or up above where the property is declared

Reply:
Hm, yeah I can do that. The name is also a bit vague in retrospect.

Reply:
Decided to keep the name. Done!